_posts/2021-07-05-changelog-150.markdown

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #150"
+date: 2021-07-05 17:00:00 +0300
+---
+
+
+## New Features
+
+* [#7373] FEATURE: Support [`:pat_param`](https://blog.rust-lang.org/2021/05/11/edition-2021.html#or-patterns-in-macro_rules) `macro_rules!` matcher (from Rust 1.53.0), FIX: Fix `:pat` matcher - now it should not parse `or patterns`
+
+* [#7362] ANN: introduce a color for `std::fmt` format specifiers
+
+* [#7356] Provide highlighting of [format string literals](https://doc.rust-lang.org/std/fmt/) in any macros that expand to macros like println, write, format, etc
+
+* [#7347] highlight *\*emphasis\** and **\*\*strong emphasis\*\*** text in rustdocs
+
+* [#7307] Support [procedural macro](https://doc.rust-lang.org/reference/procedural-macros.html) expansion on Apple Silicon machines. Note:
+- you should use `aarch64-apple-darwin` toolchain to make procedural macro expansion work
+- the proc macro expansion is still disabled by default. To turn it on, enable `org.rust.cargo.evaluate.build.scripts` and `org.rust.macros.proc` [experimental features](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-faq.html#experimental-features)
+
+* [#7211] Allow users to create their own postfix templates (by [@shocoman])
+
+* [#7171] TYPE: highlight all 'await' when under the caret (by [@t-kameyama])
+
+* [#7137] Add an intention to create a tuple struct from an unresolved function call. (by [@Kobzol])
+
+* [#7060] Added an editor intention that generates a documentation stub as demonstrated in: https://doc.rust-lang.org/beta/rust-by-example/meta/doc.html and https://github.com/rust-lang/rust/issues/57525 (by [@0x7CA])
+
+* [#6842] Extend RsSelfConvention inspection with new method prefixes (get, set and with). (by [@Kobzol])
+
+## Fixes
+
+* [#7412] Do not expand type aliases when generating getters and setters. (by [@Kobzol])
+
+* [#7407] Parenthesize inlined expressions in Inline Value refactoring if they could produce invalid or semantically different code after performing the refactoring. (by [@Kobzol])
+
+* [#7397] The intention to flatten use imports now works properly on all places inside a use group. (by [@Kobzol])
+
+* [#7395] Properly escape crate urls produced by line markers in Cargo.toml. should fix exception by clicking on the corresponding icon if you use [IntelliJ Projector](https://blog.jetbrains.com/blog/2021/03/11/projector-is-out/) with IntelliJ Rust plugin on backend
+
+* [#7393] 
```

Comment:
I'm sorry, I forget to write an actual changelog item. I think the PR description is mostly useful https://github.com/intellij-rust/intellij-rust/pull/7393